website/static/js/home-page/quickProjectSearchPlugin.js

```diff
@@ -46,7 +47,7 @@ var QuickSearchProject = {
         self.templateNodes.start();
 
         // Load up to first ten nodes
-        var url = $osf.apiV2Url('users/me/nodes/', { query : { 'embed': 'contributors'}});
+        var url = $osf.apiV2Url('users/me/nodes/', { query : { 'embed': ['contributors','root', 'parent']}});
```

Comment:
We're still loading all of the user's projects, 10 at a time. Sure, the user will see content after the first 10 load, but the user won't be able to filter their all their projects until they're all loaded, so we still want that to be relatively fast.